src/main/java/appeng/tile/storage/TileDrive.java

```diff
@@ -63,6 +63,10 @@
 public class TileDrive extends AENetworkInvTile implements IChestOrDrive, IPriorityHost
 {
 
+	private static final int BIT_POWER_MASK = 0x80000000;
+	private static final int BIT_BLINK_MASK = 0x24924924;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "BIT_BLINK_MASK" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1068)